stdlib/csv/csv_test.flux

```diff
@@ -0,0 +1,54 @@
+package csv_test
```

Comment:
Does this duplicate all the tests from the csv package? Can we delete the old ones?

Reply:
There were no tests previously in the csv package. These are net new tests. I think we were relying on the fact that almost all tests use csv parsing to work to cover the csv package. Now we have explicit tests

Reply:
"These net new tests" is the kind of patch I like to review.